nixos/modules/services/mail/protonmail-bridge.nix

```diff
@@ -0,0 +1,42 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.protonmail-bridge;
+in
+{
+
+  options.services.protonmail-bridge = {
+    enable = lib.mkEnableOption "protonmail bridge";
+
+    package = lib.mkPackageOption pkgs "protonmail-bridge" { };
+
+    path = lib.mkOption {
+      type = lib.types.listOf lib.types.path;
+      default = [ ];
+      example = lib.literalExpression "with pkgs; [ pass gnome-keyring ]";
+      description = "List of derivations to put in protonmail-bride's path.";
+    };
+
+  };
+  config = lib.mkIf cfg.enable {
+    systemd.user.services.protonmail-bridge = {
+      description = "protonmail bridge";
+      wantedBy = [ "graphical-session.target" ];
+      after = [ "graphical-session.target" ];
+
+      serviceConfig = {
+        ExecStart = "${cfg.package}/bin/protonmail-bridge --noninteractive --log-level debug";
```

Comment:
Why setting the log level to debug by default ?

Reply:
The service is really not that chatty. Would you prefer that I add a 'debugLevel' option instead and set info as the default?

Reply:
To be honest, yes. I would go further and remove the `--log-level` flag.

Reply:
Cool, will do :)

Reply:
Added `logLevel` option which defaults to null, in which case `--log-level` is not appended to the arguments when starting the service. Otherwise, if `logLevel != null`, then the service is started with `--log-level ${cfg.logLevel}`.